src/App.css

```diff
@@ -1 +1,58 @@
-/*Schrijf hier jouw eigen CSS!*/
\ No newline at end of file
+/*Schrijf hier jouw eigen CSS!*/
+
+.container {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: center;
+  min-height: 100vh;
+  padding: 2rem;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  flex: 0 0 auto;
+}
+
+header > :nth-child(2) {
+  display: flex;
+  gap: 1rem;
+}
+
+h1 {
+  font-size: 4rem;
```

Comment:
Als je gaat `rem`-en moet je altijd even de html-font declaratie gebruiken zodat het duidelijk is wat 1rem eigenlijk is en je dat gemakkelijk kunt aanpassen:

```
html {
   font-size: 16px;
}
```